manifests/s/Samsung/GalaxyBudsManager/2.1.220907.51/Samsung.GalaxyBudsManager.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.GalaxyBudsManager
+PackageVersion: 2.1.220907.51
+PackageLocale: en-US
+Publisher: Samsung Electronics Co, Ltd.
+PackageName: Samsung Galaxy Buds Manager

```

```suggestion
PackageName: GalaxyBudsManager
```
This should match Control Panel